tests/acceptance/bootstrap/WebDav.php

```diff
@@ -3711,6 +3711,42 @@ public function userDownloadsThePreviewOfWithWidthHeightProcessorUsingWebDAVAPI(
 		$this->setResponse($response);
 	}
 
+	/**
+	 * @When user :user downloads the preview of federated share image :path with width :width and height :height and processor :processor using the WebDAV API
```

Comment:
can we use existing step for downloading preview from normal share

Reply:
it's not possible to use existing steps for federation shares because federation shares are accessible only with remoteItem id (which we don't do for normal tests)

Reply:
We need the remoteItem Id of the shared image that's why i created the new step.

Using the following existing step for thumbnail would give 404 status code:
``` feature
When user "Alice" downloads the preview of "testavatar.jpg" with width "32" and height "32" and processor "thumbnail" using the WebDAV API
```